src/rut-helpers.ts

```diff
@@ -13,6 +13,10 @@ export function rutValidate(value: string) {
     return false;
   }
 
+  if (/[A-Za-z]/g.test(value.replace(/[kK]/g, ''))) {
+    return false
+  }
+
   const rut: string = rutClean(value);
```

Comment:
I would add a onliner

Reply:
> 

Sure, thought the same, but I figured I better keep the convention already used in the file.